data/mods/MindOverMatter/powers/vitakinesis_eoc.json

```
@@ -458,5 +458,103 @@
+        "type": "mixed"
+      }
+    ]
+  },
+  {
+    "id": "ACT_VITAKIN_RESTORE_LIMB_RIGHT_LEG",
+    "type": "activity_type",
+    "activity_level": "LIGHT_EXERCISE",
+    "verb": "meditating",
+    "based_on": "time",
+    "completion_eoc": "EOC_VITAKIN_RESTORE_LIMB_RIGHT_LEG_RESULT"
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_VITAKIN_RESTORE_LIMB_RIGHT_LEG_RESULT",
+    "condition": { "roll_contested": { "math": [ "rand( u_val('spell_level', 'spell: vita_limb_restore'))" ] }, "difficulty": 14 },
+    "effect": [
+      { "u_message": "Your right leg had been restored.", "type": "good" },
+      { "u_lose_trait": "NO_RIGHT_LEG" },
+      { "u_lose_var": "missing_right_leg", "type": "traits", "context": "limbs" },
+      { "u_add_trait": "HAS_RIGHT_LEG" },
+      { "run_eocs": "EOC_VITA_RESTORE_LIMB_DRAIN" },
+      { "math": [ "u_val('vitamin', 'name:vitamin_psionic_drain')", "+=", "rng( 45,75 )" ] }
+    ],
+    "false_effect": [
+      { "u_message": "Your healing powers were not great enough, you'll have to try again.", "type": "bad" },
+      { "math": [ "u_val('vitamin', 'name:vitamin_psionic_drain')", "+=", "rng( 45,75 )" ] }
+    ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_VITAKIN_RESTORE_LIMB_LEFT_LEG",
+    "condition": { "u_has_trait": "NO_LEFT_LEG" },
+    "effect": [ { "u_assign_activity": "ACT_VITAKIN_RESTORE_LIMB_LEFT_LEG", "duration": "540 minutes" } ],
+    "false_effect": [
+      {
+        "u_message": "You have a left leg or equivalent thereof.  If it's a prosthetic, please take it off before restoring your leg.",
+        "type": "mixed"
+      }
+    ]
+  },
+  {
+    "id": "ACT_VITAKIN_RESTORE_LIMB_LEFT_LEG",
+    "type": "activity_type",
+    "activity_level": "LIGHT_EXERCISE",
+    "verb": "meditating",
+    "based_on": "time",
+    "completion_eoc": "EOC_VITAKIN_RESTORE_LIMB_RIGHT_LEFT_RESULT"
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_VITAKIN_RESTORE_LIMB_LEFT_LEG_RESULT",
+    "condition": { "roll_contested": { "math": [ "rand( u_val('spell_level', 'spell: vita_limb_restore'))" ] }, "difficulty": 14 },
+    "effect": [
+      { "u_message": "Your left leg had been restored.", "type": "good" },
+      { "u_lose_trait": "NO_LEFT_LEG" },
+      { "u_lose_var": "missing_left_leg", "type": "traits", "context": "limbs" },
+      { "u_add_trait": "HAS_LEFT_LEG" },
+      { "run_eocs": "EOC_VITA_RESTORE_LIMB_DRAIN" },
+      { "math": [ "u_val('vitamin', 'name:vitamin_psionic_drain')", "+=", "rng( 45,75 )" ] }
+    ],
+    "false_effect": [
+      { "u_message": "Your healing powers were not great enough, you'll have to try again.", "type": "bad" },
+      { "math": [ "u_val('vitamin', 'name:vitamin_psionic_drain')", "+=", "rng( 45,75 )" ] }
+    ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_VITA_RESTORE_LIMB_DRAIN",
+    "//": "About 2.5 times as costly as the super heal drain.  You are regrowing a whole body part, you know.",
```

```suggestion
    "//":  "About 2.5 times as costly as the super heal drain.  You are regrowing a whole body part, you know.",
```
The way that Anabolic Rejuvenation works, it runs that drain every 30 seconds when a tick of healing occurs, whereas Ameliorate Extremity only runs the drain once. If it's intended to be 2.5 times at costly it would have to be much higher, since level 1 Anabolic Rejuvenation will tick an average of 8 times, causing the calorie/vitamin/etc drain each time.

---

Does Anabolic Rejuvenation's drain tick more than 8 times at higher levels?

---

Yes. It and the healing are set to run once every 30 seconds, so it becomes more powerful by increasing duration, which increases the total healing done and the total calorie/vitamin cost. It's 30-60 seconds (random) increased duration per level, so 1-2 ticks.

---

Alright. I ran some math on it and have increased the scaling using a level 12 spell approximation.